src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -34,30 +31,25 @@ import NotificationGroup from './NotificationGroup';
 const SelectNotificationOptionsAlert = ({ firstChannelId }) => {
   return (
     <div data-testid="select-options-alert">
-      <AlertBox
-        status={ALERT_TYPE.WARNING}
-        headline="Select your notification options"
-        level={2}
-      >
-        <div>
-          <p>
-            We’ve added notification options to your profile. Tell us how you’d
-            like us to contact you.
-          </p>
-          <p>
-            <a
-              href={`#${firstChannelId}`}
-              className="vads-u-text-decoration--none vads-u-padding--1 vads-u-margin-left--neg1 vads-u-margin-top--neg1 vads-u-margin-bottom--neg1"
-            >
-              <i
-                aria-hidden="true"
-                className="fas fa-arrow-down vads-u-margin-right--1"
-              />{' '}
-              Select your notification options
-            </a>
-          </p>
-        </div>
-      </AlertBox>
+      <va-alert status="warning">
```

Comment:
I don't know why but the AlertBox failed to re-render even though the parent `firstChannelId` was updating. But the `va-alert` works. 🤷